src/components/Login/Login.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import './Login.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faGoogle } from '@fortawesome/free-brands-svg-icons';
+import { useState } from 'react';
```

Comment:
```suggestion
import React,{useState} from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faFacebook, faGoogle } from '@fortawesome/free-brands-svg-icons';
import './Login.scss';
```
import 순서, 같은 패키지에서 import 할 수 있는 부분도 잘 확인 해 주세요

Reply:
네. scss파일 하단으로 이동 했습니다.